app-emulation/google-compute-engine/google-compute-engine-20160930.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2016 CoreOS, Inc. All rights reserved.
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+inherit eutils
+
+DESCRIPTION="Linux Guest Environment for Google Compute Engine"
+HOMEPAGE="https://github.com/GoogleCloudPlatform/compute-image-packages"
+SRC_URI="https://github.com/GoogleCloudPlatform/compute-image-packages/archive/${PV}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="amd64 x86"
+IUSE=""
+
+DEPEND="dev-python/setuptools"
+
+# These dependencies cover everything possibly called by the scripts.
+RDEPEND="
+	app-admin/sudo
+	dev-python/boto
+	dev-python/setuptools
+	sys-apps/ethtool
+	sys-apps/gawk
+	sys-apps/grep
+	sys-apps/iproute2
+	sys-apps/shadow
+	sys-libs/glibc
+	sys-libs/nss-usrfiles
+"
+
+S="${WORKDIR}/compute-image-packages-${PV}"
+
+src_prepare() {
+	# Don't attempt to mess with our host keys.
+	sed -i -e '/set_host_keys/s/true/false/i' \
```

Comment:
Instead of modifying the python code, all you need to do is create a template file, /etc/default/instance_configs.cfg.template with the following to override (you can override any of the settings like this):

[InstanceSetup]
set_host_keys = false